src/files/BrsFile.ts

```diff
@@ -121,16 +121,13 @@ export class BrsFile {
             throw new Error(`File was already processed. Create a new instance of BrsFile instead. ${this.pathAbsolute}`);
         }
 
-        //split the text into lines
-        let lines = util.getLines(fileContents);
```

Comment:
This is a nice benefit, we no longer need to split the file contents by line and process it, since `getIgnores` works on the list of tokens now.